variables.tf

```
@@ -136,3 +136,15 @@ variable "root_block_device_encrypted" {
   default     = true
   description = "Whether to encrypt the root block device"
 }
+
+variable "create_public_subnets" {
```

Cloud Posse convention is to use `*_enabled` for boolean flags
```suggestion
variable "public_subnets_enabled" {
```